src/widget/OktaSignIn.js

```diff
@@ -1,12 +1,17 @@
 /*globals module, Promise */
 import _ from 'underscore';
+import { internal } from 'okta';
 import Errors from 'util/Errors';
 import OAuth2Util from 'util/OAuth2Util';
 import Util from 'util/Util';
 import createAuthClient from 'widget/createAuthClient';
 import V1Router from 'LoginRouter';
 import V2Router from 'v2/WidgetRouter';
 
+// Enable "autoComplete" attribute
+const { FormUtil } = internal.views.forms.helpers;
+FormUtil.INPUT_OPTIONS.push('autoComplete');
```

Comment:
Why not change Courage?

Reply:
Initially, I was planning to limit the change in SIW only, but looks like the change should be safe to stay in courage. PR added here: https://github.com/okta/okta-ui/pull/3371